.github/workflows/tests.yml

```diff
@@ -133,6 +133,17 @@ jobs:
           cat bottles/steps_output.txt
           rm bottles/steps_output.txt
 
+      - name: Upload logs
+        if: always()
```

Comment:
Do we consistently use these? Even on success?

Reply:
Hmm, yeah I could probably make it fail only.

Reply:
Yeh, fail only was what I was thinking 👍. Save some time/storage all round when it's a success.